pkg/audit/verifier.go

```diff
@@ -105,6 +106,7 @@ func (d *defaultDownloader) getShare(ctx context.Context, stripeIndex, shareSize
 	if err != nil {
 		return s, err
 	}
+	defer utils.LogClose(rc)
```

Comment:
Return this error rather than logging it.

Reply:
I'm using LogClose to close the read closer, so how would I return the error instead of logging?